backend/NovuPy/activity.py

```diff
@@ -0,0 +1,45 @@
+import httpx
+
+from .settings import Core
+
+
+class Activity(Core):
+
+    async def get_activity(self, **kwargs):
+        """
+        Get activity feed
+
+        A page query parameter can be passed into the function to paginate response e.g page = 2  
+        Example response:
+
+        {
+            totalCount: 0,
+            data: ["data"],
+            pageSize: 0,
+            page: 0
+        }
+
+        """
+        url = self.base_url+'v1/activity'
+
+        async with httpx.AsyncClient() as requests:
+            response = await requests.get(url=url, headers=self.s_header, params=kwargs)
+
+        return response.json()
+
+    async def get_activity_stats(self):
+        """
+        Get activity statistics
+
+        Example response:
+        {
+            weeklySent: 0,
+            monthlySent: 0,
+            yearlySent: 0
+        }
+        """
+        url = self.base_url+'v1/activity/stats'
+        async with httpx.AsyncClient() as requests:
+            response = await requests.get(url=url, headers=self.s_header)
+
+        return response.json()
```

Comment:
why is this code repeated? same here
#140

Reply:
Actually what happened was that. I made major changes to the settings.py file and just little changes in some of the other files. I was working in another directory on my system, after the changes i just copied and replaced the same folder in this zuri-chat directory then I made a commit to same branch as before.  I was also surprised when I saw that instead of showing the new changes, it highlighted everything as “new”.